hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/TokenUpdateNftValidator.java

```diff
@@ -35,22 +26,4 @@ public class TokenUpdateNftValidator {
     public TokenUpdateNftValidator(@NonNull final TokenAttributesValidator validator) {
         this.validator = validator;
     }
-
```

Comment:
We can probably do away with this class completely now ?

Reply:
yes agreed 💯 . Will remove